tests/unit/s2n_crl_test.c

```diff
@@ -592,6 +592,15 @@ int main(int argc, char *argv[])
         EXPECT_NOT_NULL(config);
         EXPECT_SUCCESS(s2n_config_set_crl_lookup_cb(config, crl_lookup_test_callback, &data));
 
+        /* Enabling CRL validation and disabling certificate validity period validation can both
```

Comment:
Nit: This is quite a confusing paragraph. I think you're trying to say "x509 verification will succeed whether or not we have disabled validating the validity period on a certificate. This is because... _reason"_

Reply:
I simplified this to just state what is being tested.